packages/ckeditor5-dev-tests/bin/save-revision.js

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env node
+
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const branch = 'master'; // process.env.TRAVIS_BRANCH;
```

Comment:
Oops. This is checked below: https://github.com/ckeditor/ckeditor5-dev/pull/54/files/85204c96227bfe75ae5aad04cd8f3ba56bffe61f#diff-d697025760f8f24c3177d8ba07dacdaeR12 :D